client/signup/config/flows.js

```diff
@@ -87,7 +87,14 @@ const flows = {
 	},
 
 	main: {
-		steps: [ 'survey', 'design-type', 'themes', 'domains', 'plans', 'survey-user' ],
+		steps: [ 
+			'survey',
+			'design-type',
+			'themes',
+			'domains',
+			config.isEnabled( 'manage/plan-features' ) ? 'plansFeatures' : 'plans',
```

Comment:
Sorry if I wasn't clear enough earlier, I was suggesting that we modify the `plans` step rather than add `planFeatures` here. This allows up to update the four other flows that use plans. Plus @artpi's shortcut in https://github.com/Automattic/wp-calypso/pull/6451